Use pulseaudio if available or fallback to tinyalsa

akodanka opened the pull request:
Signed-off-by: akodanka <anoob.anto.kodankandath@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon_pmr0-autobuild/builds/55

akodanka committed:
Use pulseaudio if available or fallback to tinyalsa

If HAL is able to connect to the pulse audio socket, use that for all audio.
Else, fallback to the default audio HAL routine.

Tracked-On: OAM-88734
Signed-off-by: akodanka <anoob.anto.kodankandath@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon_pmr0-autobuild/builds/78

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon_pmr0-autobuild/builds/79

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon_pmr0-autobuild/builds/81

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

Cactus automatically verified this PR.

For more information, see: /absp/builders/celadon_pmr0-autobuild/builds/82

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon_pmr0-preintegration/builds/134

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

Cactus automatically verified this PR.

For more information, see: /absp/builders/celadon_pmr0-autobuild/builds/84

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon_pmr0-preintegration/builds/137

kkm123 reviewed (approved):
LGTM

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon_pmr0-preintegration/builds/138

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0-preintegration/builds/139

Done by "Kodankandath, Anoob Anto" <anoob.anto.kodankandath@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0-premerge/builds/65

Done by "Kodankandath, Anoob Anto" <anoob.anto.kodankandath@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Kodankandath, Anoob Anto" <anoob.anto.kodankandath@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon_pmr0-mergerequest/builds/66
